docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -0,0 +1,164 @@
+# Importing an existing volume
+
+Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) yaml file.
+
+Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) yaml file:
+
+-   `volumeHandle`
+-   `volumeAttributes` (optional)
```

Comment:
as of 1.10.0, `volumeAttributes` is officially no longer relevant (and since it was optional, it never was really)

Reply:
so this line should be deleted?

Reply:
and the "Including" after it, and since this list will only have 1 item, we can rephrase the preceding sentence as well

Reply:
updated in https://github.com/IBM/ibm-block-csi-driver/pull/532